packages/server/utils/makeCreateGitHubTaskComment.ts

```diff
@@ -0,0 +1,15 @@
+import {ExternalLinks} from '../../client/types/constEnums'
+
+// wrap anything that looks like an issue in ticks so it doesn't turn into a link to that issue
+const makeCreateGitHubTaskComment = (
+  creator: string,
+  assignee: string,
+  teamName: string,
+  teamDashboardUrl: string
+) =>
+  `Created by ${creator} for ${assignee}
```

Comment:
let's sanitize the creator & assignee name in case they have a # in their name followed by a number:

```
.replace(/#(\d+)/g, '#​\u200b$1')
```